Kitodo/src/main/webapp/WEB-INF/templates/includes/importConfigurationEdit/newSearchFieldDialog.xhtml

```diff
@@ -0,0 +1,91 @@
+<!--
+ *
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ *
+-->
+
+<ui:composition
+        xmlns="http://www.w3.org/1999/xhtml"
+        xmlns:h="http://xmlns.jcp.org/jsf/html"
+        xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+        xmlns:p="http://primefaces.org/ui">
+    <p:dialog id="addSearchFieldDialog"
+              width="640"
+              resizable="false"
+              modal="true"
+              dynamic="true"
+              widgetVar="addSearchFieldDialog">
+        <h:form id="addSearchFieldForm">
+            <h3>#{msgs['importConfig.addSearchField']}</h3>
+            <h:panelGroup layout="block"
+                          styleClass="dialogFieldWrapper">
+                <p:panelGrid columns="1" layout="grid">
+                    <p:row>
+                        <div>
+                            <p:outputLabel for="searchFieldLabel"
+                                           value="Search field label"/>
+                            <p:inputText id="searchFieldLabel"
+                                         styleClass="input-with-button"
+                                         value="#{addSearchFieldDialogView.searchField.label}"/>
+                            <p:commandButton id="searchFieldLabelHelp" type="button"
+                                             styleClass="help-button" icon="fa fa-lg fa-question-circle-o"/>
+                            <p:tooltip for="searchFieldLabelHelp"
+                                       value="Label displayed to the user when importing processes using the current OPAC import configuration"/>
+                        </div>
+                    </p:row>
+                    <p:row>
+                        <div>
+                            <p:outputLabel for="searchFieldValue" value="Search field value"/>
+                            <p:inputText id="searchFieldValue"
+                                         styleClass="input-with-button"
+                                         value="#{addSearchFieldDialogView.searchField.value}"/>
+                            <p:commandButton id="searchFieldValueHelp" type="button"
+                                             styleClass="help-button" icon="fa fa-lg fa-question-circle-o"/>
+                            <p:tooltip for="searchFieldValueHelp"
```

Comment:
Please add i18n keys for all tooltip messages.

Reply:
see above